src/components/Loader/pages/Navigation.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => {
+  return (
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          to="/"
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'has-background-grey-lighter': isActive },
+          )}
+        >
+          Home
+        </NavLink>
+
+        <NavLink
+          to="/people"
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'has-background-grey-lighter': isActive },
+          )}
+        >
+          People
+        </NavLink>
```

Comment:
```suggestion
        <NavLink
          to="/"
          className={({ isActive }) => classNames(
            'navbar-item',
            { 'has-background-grey-lighter': isActive },
          )}
        >
          Home
        </NavLink>

        <NavLink
          to="/people"
          className={({ isActive }) => classNames(
            'navbar-item',
            { 'has-background-grey-lighter': isActive },
          )}
        >
          People
        </NavLink>
```

you can create separate component for this case, Misha said that it's a good practice